internal/command/errors/errors.go

```diff
@@ -0,0 +1,10 @@
+package errors
+
+import (
+	"github.com/spf13/cobra"
+	sentry_ext "github.com/superfly/flyctl/internal/command/extensions/sentry"
+)
+
+func New() (cmd *cobra.Command) {
+	return sentry_ext.Dashboard()
```

Comment:
wdyt about extending `fly open` to `fly open sentry` or `fly open errors` instead of creating another toplevel command for what is an already polluted command root.

Another option is `fly app errors`

Reply:
I don't have strong feelings here. I'll move to `apps errors` for now.

Reply:
For now I just removed this until the aliasing can be done in a cleaner fashion. Now aliases have unfortunate help output.